engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -1030,12 +1030,13 @@ namespace dmGameSystem
         return (dmGameObject::HComponent)params.m_UserData;
     }
 
+    // we can use only 24 bits, take a look render_private.h -> RenderListSortValue
```

Comment:
This is a bit hidden knowlege, So I thought I can use all 32 bits what I did here: https://github.com/defold/defold/pull/10631/files